modules/react/src/components/export-video/export-video-panel-preview.js

```diff
@@ -148,46 +150,35 @@ export class ExportVideoPanelPreview extends Component {
     // Adds mapbox layer to modal
     const map = this.mapRef.current.getMap();
     const deck = this.deckRef.current.deck;
+
+    const layerOrder = this.props.mapData.visState.layerOrder;
+    const keplerLayers = layerOrder
+      .slice()
+      .reverse()
+      .reduce(this._renderLayer, []);
+
     map.addLayer(new MapboxLayer({id: 'my-deck', deck}));
-    // TODO trying to make map scale to 100% of modal
-    // map.on('load', function () {
-    //   map.resize();
-    // });
+
+    for (let i = 0; i < keplerLayers.length; i++) {
+      // Adds DeckGL layers to Mapbox so Mapbox can be the bottom layer. Removing this clips DeckGL layers
+      map.addLayer(new MapboxLayer({id: keplerLayers[i].id, deck}));
+    }
+
     map.on('render', () =>
       this.props.adapter.onAfterRender(() => {
         this.forceUpdate();
       })
     );
-    map.resize();
```

Comment:
While we seem to don't need `map.resize();` anymore, I'm making a note that this has been removed.

Reply:
It's redundant as well as resize() is also called in the [_resizeVideo function](https://github.com/uber/hubble.gl/blob/f74d827ff4ac0abb6627f6b25765ad3177c18a29/modules/react/src/components/export-video/export-video-panel-preview.js#L70)